plugins/hasura/hasura.plugin.zsh

```diff
@@ -0,0 +1,13 @@
+if (( ! $+commands[hasura] )); then
+  return
+fi
+
+# If the completion file does not exist, generate it and then source it
+# Otherwise, source it and regenerate in the background
+if [[ ! -f "$ZSH_CACHE_DIR/completions/_hasura" ]]; then
+  helm completion zsh --file "$ZSH_CACHE_DIR/completions/_hasura" >/dev/null
```

Comment:
Is `helm` supposed to provide `hasura` completions? I don't really know any of the projects 😅

Reply:
🤦  Fixed to reference the `hasura` cmd